ci-operator/config/containers/libpod/containers-libpod-v1.6.yaml

```diff
@@ -0,0 +1,52 @@
+base_images:
+  base:
+    name: "4.4"
+    namespace: ocp
+    tag: base
+  base-machine:
+    cluster: https://api.ci.openshift.org
```

Comment:
Could you please do the cluster for other base_images too?

Reply:
Could you point me to how I can do that?

Reply:
He means -- add this `cluster: https...` field to every item in `base_images`

Reply:
Ah got it! Fixed :)